lib/shell.js

```diff
@@ -45,6 +45,7 @@ const async = require('async'),
                         execOption.pty = true;
                     }
 
+                    runCommand = "source /etc/profile;" + runCommand;
```

Comment:
could you replace ";" to "&&" ?  
 runCommand = "source /etc/profile && " + runCommand;

Please refer to ares-cli/lib/ares-install.js 
let cmd = '/bin/rm -rf ' + config.tempDirForIpk + ' && /bin/mkdir -p ' + config.tempDirForIpk;

Reply:
done